examples/proc_combined_ODNP.py

```diff
@@ -264,5 +264,5 @@
     fl.plot(np.real(s_int['power',:-3]),'ko',capsize=6,alpha=0.3)
     fl.plot(np.real(s_int['power',-3:]),'ro',capsize=6,alpha=0.3)
     #}}}
-fl.show();quit()
```

Comment:
Again JUST removing quit statement as JF saw it when he searched "quit". unrelated to this PR and so not touched aside from removing quit